lx_administration/autoconf/imports/utils.py

```
@@ -91,12 +85,103 @@ def load_group_vars(group_vars_dir: Path) -> Dict[str, str]:
-        host_name = host_vars_file.stem
-        host_vars[host_name] = _host_vars
+            if host_name in host_vars:
+                host_vars[host_name] = deep_update(host_vars[host_name], _host_vars)
+            else:
+                host_vars[host_name] = _host_vars
 
     return host_vars
+
+def load_home_host_vars(ansible_inventory_dir: Path) -> Dict[str, dict]:
+    from lx_administration.autoconf.imports.utils import _load_config, deep_update
+    import pprint
```

Ruff (0.11.9)

`pprint` imported but unused

Remove unused import: `pprint`

(F401)